### PR TITLE
HBASE-25354 Update create-release scripts; rotted

### PR DESCRIPTION
**Pull request by saintstack**
dev-support/create-release/do-release.sh
   move gpg check to non-docker context. Also cleanup tmp files when done.

  dev-support/create-release/hbase-rm/Dockerfile
  dev-support/create-release/mac-sshd-gpg-agent/Dockerfile
   Hack to update packages... the old ones  no longer available.

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  9s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 22s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  7s |  Maven dependency ordering for patch  |
||| _ Other Tests _ |
|  |   |   2m 30s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.40 ServerAPI=1.40 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2734/1/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2734 |
| Optional Tests |  |
| uname | Linux fbeb34a1b972 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 946fa81715 |
| Max. process+thread count | 51 (vs. ulimit of 30000) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2734/1/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   3m 59s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 23s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  8s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  hadolint  |   0m  3s |  There were no new hadolint issues.  |
| -0 :warning: |  shellcheck  |   0m  2s |  The patch generated 7 new + 9 unchanged - 0 fixed = 16 total (was 9)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
||| _ Other Tests _ |
| +0 :ok: |  asflicense  |   0m  0s |  ASF License check generated no output?  |
|  |   |   5m 39s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.40 ServerAPI=1.40 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2734/1/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2734 |
| Optional Tests | dupname asflicense shellcheck shelldocs hadolint |
| uname | Linux fe464d0640ca 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 946fa81715 |
| shellcheck | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2734/1/artifact/yetus-general-check/output/diff-patch-shellcheck.txt |
| Max. process+thread count | 52 (vs. ulimit of 30000) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2734/1/console |
| versions | git=2.17.1 maven=3.6.3 shellcheck=0.4.6 hadolint=1.17.5-0-g443423c |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   4m 25s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 25s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  8s |  Maven dependency ordering for patch  |
||| _ Other Tests _ |
|  |   |   5m 52s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.40 ServerAPI=1.40 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2734/1/artifact/yetus-jdk8-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2734 |
| Optional Tests |  |
| uname | Linux 703cdcaf3168 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 946fa81715 |
| Max. process+thread count | 51 (vs. ulimit of 30000) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2734/1/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Review by Apache9 — APPROVED** [review]
Overall LGTM. The only thing is for the openjdk-8-jdk, as we already have the version in the package name(openjdk-'8'), do we still need to set the version? Just use the newest one is enough?

File: dev-support/create-release/hbase-rm/Dockerfile [hunk 1]
```diff
@@ -34,7 +34,7 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -qq -y update \
     libxml2-dev='2.9.4+dfsg1-*' \
     lsof='4.89+dfsg-*' \
     maven='3.6.0-*' \
-    openjdk-8-jdk='8u252-b09-*' \
+    openjdk-8-jdk='8u275-b01-*' \
```
[the remark this answers is not in the document]
Comment (virajjasani) [reply dev-support/create-release/hbase-rm/Dockerfile:37]: I used `8u*` locally while creating previous RCs as `8u252-b09-*` was anyways unavailable:
```
E: Version '8u252-b09-*' for 'openjdk-8-jdk' was not found
```

[the remark this answers is not in the document]
Comment (busbey) [reply dev-support/create-release/hbase-rm/Dockerfile:37]: The version pinning was part of an attempt to get us more repeatable release builds. Unfortunately I don't think it's compatible with the infra pieces we're relying on.

Maybe instead we should use the most general version of the dependencies we need and then store generate docker images in a registry? That way we could tag the specific image used when making RCs so they could be used again later.

**Review by busbey — COMMENTED** [review]
Why the move of gpg checking outside of docker? We need a check from in the container to make sure the forwarding is working before doing the expensive build steps

File: dev-support/create-release/hbase-rm/Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (saintstack) [reply dev-support/create-release/hbase-rm/Dockerfile:37]: For the jdk, is 8u* too general? Maybe no harm being explicit about what version of jdk we build with.

We could go the busbey registry route. That'd be most stable. This build script stuff doesn't seem to be used enough though?

[the remark this answers is not in the document]
Comment (busbey) [reply dev-support/create-release/hbase-rm/Dockerfile:37]: There was a to-do somewhere about updating our nightly test that walked through the RM steps in the guide to use this stuff instead. I'd guess that hasn't happened yet.

**Comment by saintstack** [comment]
I moved the gpg agent stuff because it was a new addition failing in the docker context. I wanted to see what happened when we went to sign (This PR is not done... I should have marked it so).

Thanks for the reviews.... Let me integrate the suggestions.

File: dev-support/create-release/hbase-rm/Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (saintstack) [reply dev-support/create-release/hbase-rm/Dockerfile:37]: Man. Yeah, should do this. Then it would be doing the work here registering build images.

**Comment by saintstack** [comment]
@busbey On the gpg stuff, this is what I was getting.... Looking more (at README), I must be doing something wrong.... You have an idea? Here is what I get when I put the signing check back and when I run in docker context...

```
docker run --rm -ti --env-file /Users/stack/Downloads/2.4.0//env.list --cidfile /Users/stack/Downloads/2.4.0//release.cid --mount type=bind,src=/Users/stack/Downloads/2.4.0/,dst=/home/stack/hbase-rm,consistency=delegated --mount type=volume,src=gpgagent,dst=/home/stack/.gnupg/ org.apache.hbase/hbase-rm:latest
Enter passphrase for key '/Users/stack/.ssh/id_rsa': GPG Version: gpg (GnuPG) 2.2.4
libgcrypt 1.8.1
Copyright (C) 2017 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Home: /home/stack/.gnupg
Supported algorithms:
Pubkey: RSA, ELG, DSA, ECDH, ECDSA, EDDSA
Cipher: IDEA, 3DES, CAST5, BLOWFISH, AES, AES192, AES256, TWOFISH,
        CAMELLIA128, CAMELLIA192, CAMELLIA256
Hash: SHA1, RIPEMD160, SHA256, SHA384, SHA512, SHA224
Compression: Uncompressed, ZIP, ZLIB, BZIP2
gpg: key DB9D313DA7874F29: "Michael Stack <stack@duboce.net>" not changed
gpg: Total number processed: 1
gpg:              unchanged: 1
Testing gpg signing gpg --no-autostart --batch --local-user 0xA7874F29 --detach --armor --sign /tmp/tmp.lPAIy9ngub/gpg_test.1.txt
gpg: no gpg-agent running in this session
gpg: keydb_search failed: No agent running
gpg: skipped "0xA7874F29": No agent running
gpg: signing failed: No agent running
Trying to sign a test file using your GPG setup failed.

Please make sure you have a local gpg-agent running with access to your secret keys prior to
starting a release build. If you are creating release artifacts on a remote machine please check
that you have set up ssh forwarding to the gpg-agent extra socket.
```

Agent seems to be running fine before build.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 28s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 21s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  7s |  Maven dependency ordering for patch  |
||| _ Other Tests _ |
|  |   |   1m 54s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.40 ServerAPI=1.40 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2734/2/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2734 |
| Optional Tests |  |
| uname | Linux 19928825072d 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / f879d45fb0 |
| Max. process+thread count | 55 (vs. ulimit of 30000) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2734/2/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 33s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 24s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  7s |  Maven dependency ordering for patch  |
||| _ Other Tests _ |
|  |   |   2m  2s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.40 ServerAPI=1.40 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2734/2/artifact/yetus-jdk8-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2734 |
| Optional Tests |  |
| uname | Linux e6c75eba673a 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / f879d45fb0 |
| Max. process+thread count | 52 (vs. ulimit of 30000) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2734/2/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 33s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  1s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  1s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 25s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  8s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  hadolint  |   0m  2s |  There were no new hadolint issues.  |
| -0 :warning: |  shellcheck  |   0m  4s |  The patch generated 7 new + 9 unchanged - 0 fixed = 16 total (was 9)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
||| _ Other Tests _ |
| +0 :ok: |  asflicense  |   0m  0s |  ASF License check generated no output?  |
|  |   |   2m 24s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.40 ServerAPI=1.40 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2734/2/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2734 |
| Optional Tests | dupname asflicense shellcheck shelldocs hadolint |
| uname | Linux 4250c3fcdd61 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / f879d45fb0 |
| shellcheck | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2734/2/artifact/yetus-general-check/output/diff-patch-shellcheck.txt |
| Max. process+thread count | 52 (vs. ulimit of 30000) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2734/2/console |
| versions | git=2.17.1 maven=3.6.3 shellcheck=0.4.6 hadolint=1.17.5-0-g443423c |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by saintstack** [comment]
@busbey The above might be me. I got further. Seems like I need ssh-agent up too. The new push puts the gpg test back for the docker context and seems to work.

**Comment by busbey** [comment]
what's the build output say about trying to launch the container for the gpg proxy? on a mac build it has to launch a container first that can receive ssh connections and then it launches an ssh tunnel to the gpg agent's socket.

This is the bit of launching the release container that is supposed to be mounting a shared volume that can see the forwarded gpg agent socket: `--mount type=volume,src=gpgagent,dst=/home/stack/.gnupg/`. If that isn't working my guess is the proxy container failed to build. (but that should fail the whole affair)

**Comment by saintstack** [comment]
Thanks @busbey . I went back to the README, cleared out my gpg instance, and then I got further. It looks like I have to have ssh-agent up too w/ my key added so the ssh works w/o password too... Added that to the doc. Doing these two things I am further (failures because not enough mx for the local container... all fixable). Am testing running complete builds now... Thanks.

**Comment by saintstack** [comment]
Pushed an update. Adds to the README note on ssh-agent and allows that there may be no JIRAs in release (good for testing or good for scenario where we want to make a release with no changes -- the release creation broke in the middle and need to rerun)

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 34s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 23s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  7s |  Maven dependency ordering for patch  |
||| _ Other Tests _ |
|  |   |   1m 59s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.40 ServerAPI=1.40 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2734/3/artifact/yetus-jdk8-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2734 |
| Optional Tests |  |
| uname | Linux fc2e1a66bd0d 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 8634428724 |
| Max. process+thread count | 47 (vs. ulimit of 30000) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2734/3/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 32s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 12s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  8s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  hadolint  |   0m  2s |  There were no new hadolint issues.  |
| -0 :warning: |  shellcheck  |   0m  3s |  The patch generated 7 new + 9 unchanged - 0 fixed = 16 total (was 9)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
||| _ Other Tests _ |
| +0 :ok: |  asflicense  |   0m  0s |  ASF License check generated no output?  |
|  |   |   2m  4s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.40 ServerAPI=1.40 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2734/3/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2734 |
| Optional Tests | dupname asflicense shellcheck shelldocs hadolint |
| uname | Linux 527ea514129c 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 8634428724 |
| shellcheck | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2734/3/artifact/yetus-general-check/output/diff-patch-shellcheck.txt |
| Max. process+thread count | 48 (vs. ulimit of 30000) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2734/3/console |
| versions | git=2.17.1 maven=3.6.3 shellcheck=0.4.6 hadolint=1.17.5-0-g443423c |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  8s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  4s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 12s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  7s |  Maven dependency ordering for patch  |
||| _ Other Tests _ |
|  |   |   2m 24s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.40 ServerAPI=1.40 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2734/3/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2734 |
| Optional Tests |  |
| uname | Linux ae67a01b860c 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 8634428724 |
| Max. process+thread count | 45 (vs. ulimit of 30000) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2734/3/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by saintstack** [comment]
I changed the jdk8 version so its what @virajjasani suggested -- i.e. 8u* -- which is sort of what @Apache9 was suggesting. This stuff works for a dry run. Let me push what I have here so its available in case Andrew wants it for a new RC.

**Comment by saintstack** [comment]
Manually merged. Closing

**Pull request closed without merging**